src/environments/environment.ts

```diff
@@ -1,9 +1,8 @@
 import { Configuration } from "@helpers/app-initializer/app-initializer";
-import { version } from "../../package.json";
 
-export const environment: Configuration = {
+export const environment = ({
   production: false,
-  version,
-} as any;
+  version: "dev",
```

Comment:
Might add a note that this value gets templated so it doesn't get moved/formatted into a manner that sed no longer recognises.

Or:
```suggestion
  version: "<<REPLACE_WHEN_BUILT>>",
```

use a token that makes it easy to find/replace, then your sed regex can be less complex and work in more cases.